### PR TITLE
Detail page

### PR DESCRIPTION
**Pull request by twill81**
almost finished design - only 1-2 features missing. please have a look :)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,46 @@
 import { useEffect, useState } from 'react'
 import AddGamePage from './pages/AddGamePage'
+import GameDetails from './pages/GameDetails'
 import { setLocalStorage, getLocalStorage } from './utils/localStorage'
+import { Route, Switch, useHistory } from 'react-router-dom'
```
Comment (frederikebruendermann) [inline src/App.js:5]: Maybe you could organize your imports with cmd+shift+p

File: src/components/Button.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const Button = styled.button`
   padding: 20px;
   margin: 20px;
   width: 332px;
-  background-color: #c3073f;
+  background-color: #00ff00;
   color: #1a1a1d;
```
Comment (frederikebruendermann) [inline src/components/Button.js:11]: Maybe you could add GlobalStyles to your app, to have a better control on your styles.

File: src/components/RemoveButton.js [hunk 3]
```diff
@@ -3,21 +3,29 @@ import TrashIcon from '../images/trash_icon.png'
 
 export default function RemoveButton({ onRemove, id }) {
   return (
-    <Button onClick={() => onRemove(id)}>
+    <Button
+      onClick={event => {
+        event.stopPropagation()
+        onRemove(id)
+      }}
```
Comment (frederikebruendermann) [inline src/components/RemoveButton.js:10]: 🚀

File: src/components/Search.js [hunk 4]
```diff
@@ -8,48 +8,72 @@ export default function Search({ onSubmit }) {
...
         </Suggestions>
       </Form>
     </>
   )
+
+  function handleClickOption(option) {
+    setSelectedOption(option)
+    setIsSelected(true)
+    setQuery(false)
+  }
```
Comment (frederikebruendermann) [inline src/components/Search.js:68]: well done!

File: src/pages/AddGamePage.js [hunk 5]
```diff
@@ -1,14 +1,29 @@
...
+export default function AddGamePage({
+  onSubmit,
+  games,
+  onRemove,
+  handleShowDetails,
+}) {
   return (
     <Container>
+      <LogoContainer>
+        <img src={Logo} alt="" />
```
Comment (frederikebruendermann) [inline src/pages/AddGamePage.js:17]: please consider using cumulative Image Shift with your pictures :) --> https://www.smashingmagazine.com/2021/06/how-to-fix-cumulative-layout-shift-issues/

File: src/pages/GameDetails.js [hunk 6]
```diff
@@ -1,5 +1,246 @@
...
+            key={screenshot.id}
+          ></SingleScreenshot>
+        ))}
+      </Screenshots>
+
+      <DescriptionWrapper>
+        <h4>About:</h4>
+        <Description
+          dangerouslySetInnerHTML={{
+            __html: DOMpurify.sanitize(game.description),
```
Comment (frederikebruendermann) [inline src/pages/GameDetails.js:71]: That was new to me :) 👍🏼

**Review by frederikebruendermann — APPROVED** [review]
Well done! 🚀 Your code is readable. There are a few things I commented and suggestest, maybe there´s something interesting for you :) It would be a good way to define GlobalStyles for your app to make it easier to style.

File: src/components/Button.js (hunk 2 above)
Comment (twill81) [reply src/components/Button.js:11]: good point! will do. :)

**Pull request merged**